assets/queries/openAPI/invalid_schema_external_documentation_url/query.rego

```diff
@@ -103,6 +104,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	url := doc.components.responses[r].content[c].schema.externalDocs.url
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)